README.md

```diff
@@ -370,6 +372,22 @@ When you start Azurite, pass those files to the `--cert` and `--key` parameters
 azurite --cert 127.0.0.1.pem --key 127.0.0.1-key.pem
 ```
 
+#### Dotnet dev-certs
+
+You can generate a local certificate for localhost with `dotnet dev-certs`. See the [blob](https://www.hanselman.com/blog/DevelopingLocallyWithASPNETCoreUnderHTTPSSSLAndSelfSignedCerts.aspx) for detail, and then run the following commands.
+
+```bash
+dotnet dev-certs https -ep 127.0.0.1.pfx -p <your password>
```

Comment:
Can you get this to work with the default `localhost` cert please?

Reply:
Azurite run on localhost or 127.0.0.1 not controled by cert name. During our test, we can give any name to the cert, and with that cert, Azurite can run on either localhost or 127.0.0.1. If want to run against localhost, need add two more para: --blobHost, --queueHost. Command like: azurite --cert <CertName>.pem --key <CertName>-key.pem --blobHost localhost --queueHost localhost